app.py

```diff
@@ -80,11 +85,13 @@ def serve(path):
 
     @app.route("/manager/ec2_action", methods=["POST"])
     @authenticated(ADMINS_GROUP)
+    @csrf_needed
```

Comment:
suggestion: would it be safer to implement a whitelist instead? For instane, CSRF could be needed by default, and "unprotected" endpoints need to be explicitly decorated

This would also be a future improvement for the `authenticated` decorator

Reply:
I like the approach and we can definitely address this in the near future along the authentication guards